requirements-python.txt

```diff
@@ -15,13 +17,13 @@ apache-libcloud==2.3.0
 requests==2.20.0
 elasticsearch==6.0.0
 sortedcontainers==1.5.9
-jinja2==2.10
+jinja2==2.10.1
 prometheus_client==0.3.1
 enum==0.4.7
 pre-commit==1.14.4
 anyconfig==0.9.8
 click==7.0
 click-completion==0.5.0
 PTable==0.9.2
-pycrypto==2.6.1
+pycryptodome==3.8.1
```

Comment:
did the package name change?

Reply:
pycrypto isn't maintained anymore and pycryptodome is a dropin replacement for it.